Topic 0/src/com/globant/topic0/exercise1/ex2/Book.java

```diff
@@ -0,0 +1,73 @@
+package com.globant.topic0.exercise1.ex2;
+
+import java.util.Arrays;
+
+public class Book {
+
+	// Fields
+	private String name;
+	private Author[] authors;
```

Comment:
We can use another data structure instead of arrays, i.e. java.util.List (and check out any implementation).